gc/gc-tool/src/test/java/org/projectnessie/gc/tool/TestCLI.java

```diff
@@ -77,6 +81,26 @@ static void setNessieUri(@NessieClientUri URI uri) {
     nessieUri = uri;
   }
 
+  private static DataSource dataSource;
```

Comment:
Please add an `@AfterAll` and close the `dataSource` - the Agroal DSs are closeable.